1449 improve examples (#1661)
open-contracting/standard: docs/primer/releases_and_records.md

* Publish each release and record at its own persistent URL
* Produce bulk packages of releases and records for users to download

```{note}

> For guidance on how to generate releases and records see the worked examples in [merging](../schema/merging.md) and [change history](../guidance/build/change_history.md).

Reviewer: The purpose of this page is to explain the concepts of releases and records so I think that the purpose of linking to the worked examples is to illustrate the concepts rather than to explain how to implement releases and records.

```suggestion
To see what releases and records look like, refer to the worked examples in [merging](../schema/merging.md) and [change history](../guidance/build/change_history.md).
```
